src/patternfly/components/TextInputGroup/text-input-group.scss

```
@@ -61,6 +61,17 @@
 
     pointer-events: none;
   }
+
+  &.pf-m-plain {
+    --pf-c-text-input-group--backgroundcolor: transparent;
```

css vars are case sensitive

```suggestion
    --pf-c-text-input-group--BackgroundColor: transparent;
```